### ISSUE TITLE
Doesn't work with uchardet.

### ISSUE THREAD

**Issue by tayei1997** [podofo/podofo#121]
Hi, when I use podofo and uchardet(https://github.com/BYVoid/uchardet，uchardet is an encoding detector library) in the same C++ project, it reports error "LNK1000: Internal error during LIB::Search ", but with other libraries it works.

Below is the code I used to test this, the project is already configured with the relevant libraries.
`
   
      
      #include <podofo/podofo.h>
      
      #include <uchardet.h>
      
      #pragma comment(lib,"ws2_32.lib")
      
      int main() {
      
	      PoDoFo::PdfMemDocument a;
      
	      a.Load("C:\\mytest\\NewPDF5.pdf");
      
	      //uchardet_t ud = uchardet_new();
      }
`

**Comment by ceztko** [comment]
You should mention which toolsets you are using: I understand you are on Windows, but which VS version? And are you using static or dynamic linking? Anyway, these kind of issues are hard to debug since it's an internal linker error: it could be some incompatible linker options, or outdated compiler. Honestly, I'm not interested in investigating interaction of PoDoFo with <random> library, because I think in such situation if there is an issue within PoDoFo the burden of proof is yours. If you can post a minimal reproduction package (few MBs) it may help but don't expect a prompt analysis. In general the recommendations are:
- Upgrade your toolsets to last version;
- Use dynamic linking, if possible;
- Keep compilation/linking options simple.

**Comment by tayei1997** [comment]
Hi, after some troubleshooting, I've determined the cause of the conflict: I compiled the podofo library on one computer, and the uchardet library on another, and using both podofo.lib and uchardet.lib compiled on different computers causes this problem. When I compile using the same computer, the two libraries no longer conflict. 
podofo doesn't have this problem when used with other 3-way libraries I use, I'm guessing it's due to the fact that both contain character encoding.
